conftest.py

```diff
@@ -2,7 +2,7 @@
 
 import os
 
-from hypothesis import settings, Verbosity
+from hypothesis import Verbosity, settings
```

Comment:
This doesn't change anything right?

Reply:
Correct.  The only difference is in style.